src/etc/inc/system.inc

```diff
@@ -1644,13 +1644,20 @@ function system_ntp_configure() {
 	$ntpcfg .= "tinker panic 0 \n";
 
 	/* Add Orphan mode */
-	$ntpcfg .= "# Orphan mode stratum\n";
+	$ntpcfg .= "# Orphan mode stratum and Maximum candidate NTP peers\n";
 	$ntpcfg .= 'tos orphan ';
 	if (!empty($config['ntpd']['orphan'])) {
 		$ntpcfg .= $config['ntpd']['orphan'];
 	} else {
 		$ntpcfg .= '12';
 	}
+	/* Add Maximum candidate NTP peers */
+	if (!empty($config['ntpd']['ntpmaxpeers'])) {
+		$ntpcfg .= ' maxclock ';
```

Comment:
Rather than repeating `' maxclock '` in both branches of this test since it's added either way, add it to the string before the `if`, and then add either the user-supplied value or `5`. Similar to what is done just above for `orphan`.